app/utilities/styles.js

```diff
@@ -30,7 +30,7 @@ export const getStyles = el => {
 
   const { borderColor, borderWidth, borderStyle } = borders
 
-  if (parseInt(borderWidth) > 0) {
+  if (parseFloat(borderWidth) > 0) {
```

Comment:
awesome catch and contribution!

![Screenshot 2023-04-04 3 01 14 PM](https://user-images.githubusercontent.com/1134620/229932562-4c08d2ac-c527-4ebc-a068-577b097251f6.png)

Reply:
Thanks a lot!